dedibox-console/my-account/how-to/create-a-dedibox-account.mdx

```diff
@@ -7,24 +7,21 @@ content:
   paragraph: This page explains how to create a Scaleway Dedibox account
 tags: dedibox account create signup
 dates:
-  validation: 2023-01-16
+  validation: 2023-07-24
   posted: 2020-03-12
 categories:
   - account
 ---
 
-1. Open the [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/) website in a web browser.
-2. Click **Sign up** in the top right corner of the website:
-    <Lightbox src="scaleway-dedibox-signup.webp" alt="" />
-3. Fill in the required information, validate the terms of service, and click **Create my account**. A link is sent to your email address.
-    <Lightbox src="scaleway-dedibox-subscribe.webp" alt="" />
-4. Click the link to validate your email address and to activate your account. The [Dedibox Console](https://console.online.net) login screen displays.
-5. Enter your chosen username and password and click **Sign In**:
-    <Lightbox src="scaleway-dedibox-login.webp" alt="" />
+Follow these steps to get started with Scaleway Dedibox:
 
-You are now connected to the Scaleway Dedibox Console and you can order your first dedicated server:
-
-<Lightbox src="scaleway-dedibox-console.webp" alt="" />
+1. Go to the [Scaleway Dedibox](https://www.scaleway.com/en/dedibox/) website using your web browser.
+2. Locate and click the **Create Dedibox Account** button situated at the top right corner of the website.
+3. Fill in all the required information, review and accept the terms of service, and then proceed by clicking **Create my account**. A verification link will be sent to the email address you provided during registration.
+4. Access your email inbox and click on the verification link to activate your account. This will direct you to the [Dedibox Console](https://console.online.net) login screen.
```

Comment:
```suggestion
4. Access your email inbox and click on the verification link to activate your account. This will direct you to the [Dedibox console](https://console.online.net) login screen.
```